examples/spark2/src/main/scala/org/apache/carbondata/examples/StreamExample.scala

```diff
@@ -169,7 +168,7 @@ object StreamExample {
             .format("carbondata")
             .trigger(ProcessingTime("5 seconds"))
             .option("checkpointLocation", tablePath.getStreamingCheckpointDir)
-            .option("tablePath", tablePath.getPath)
+            .option("dbName", "default")
```

Comment:
It can't work if `dbName` is not specified?

Reply:
if using default database, it can work. If not, it can't work